{{ cookiecutter.repo_name }}/docker/requirements.txt

```diff
@@ -0,0 +1 @@
+mypy==0.790
```

Comment:
We should push this upstream into the Orbyter image and keep the requirements here clean.

Reply:
You aren't going to let me be lazy are you :-) ? Luckily, I was able to A.D. to add mypy to orbyter:3.2

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/12516153/103563385-884dc080-4e71-11eb-878f-e71bf73dec0f.png">